src/pybind/mgr/dashboard/requirements-test.txt

```diff
@@ -1,4 +1,3 @@
-mock; python_version <= '3.3'
 pytest<4
```

Comment:
Probably we would want to remove the `pytest<4` restriction as well? I don't remember exactly where the breaking changes came from pytest.